src/lib/config.js

```diff
@@ -22,7 +22,8 @@ const defaultConfig = {
 	getConsentDataTimeout: 500,
 	purposeOneTreatment: false,
 	disableLanguageSelect: false,
-	disableTabs: false
+	reduceStepsToConfigureConsent: false,
```

Comment:
Maybe "reduceSteps" will be enough? It is easy to make mistake in configuration panel when property name is too long.